### ISSUE TITLE
fips ubuntu pro: postinst isn't emitting a reboot notice message on upgrade (SC-3)

### ISSUE THREAD

**Issue by blackboxsw** [canonical/ubuntu-advantage-client#1390]
When upgrading a Ubuntu FIPS PRO image to version 27. Reboot required notices banners are absent, even though the `/var/lib/ubuntu-advantage/marker-reboot-cmds-required` is properly created by the[ postinst logic](https://github.com/canonical/ubuntu-advantage-client/blob/master/debian/postinst#L146-L157)

Expected behavoir would emit the message to `ua status`
```bash
SERVICE       ENTITLED  STATUS    DESCRIPTION
esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
fips          yes       enabled   NIST-certified FIPS modules
fips-updates  no        —         Uncertified security updates to FIPS modules
livepatch     yes       n/a       Canonical Livepatch service

NOTICES:
FIPS support requires system reboot to complete configuration.

Enable services with: ua enable <service>

                Account: 329910648901
           Subscription: 329910648901
            Valid until: 9999-12-31 00:00:00
Technical support level: essential
```

We are missing that NOTICES section

**Comment by blackboxsw** [comment]
This bug is invalid on AWS PRO already has 26.3 and did not have apt holds in place, I just confirmed that no `/var/lib/ubuntu-advantage/marker-reboot-cmds-required ` marker file is created because we don't have apt holds so no NOTICES section is expected in this case. 
We also have logic in postinst currently  that always [emits the notice if mark_reboot_cmds_as_needed is called](https://github.com/canonical/ubuntu-advantage-client/blob/main/debian/ubuntu-advantage-tools.postinst#L260-L266)

Closing this as fixed in ccee970c88998be5c417a894bc01e32d3baa3f2a

**Comment by blackboxsw** [comment]
This issue will still affect old AWS PRO FIPS images which contain ubuntu-advantage tools 24.3.
Those images will not properly report the "FIPS support requires system reboot to complete configuration." even though the marker file is created. 

VMs based on  `ami-0376494d8f4f09187` which contained ubuntu-advantage-tools 24.3 will not present that reboot notice message and will continue to have stale/held fips packages preventing upgrade to latest FIPS security updates.
